WordPress/gutenberg
# [RNMobile] Upgrade CI to use Xcode 12

## Description


Fixes https://github.com/WordPress/gutenberg/issues/26188

## How has this been tested?




- Check if all GitHub Actions workflows are green.
- Run e2e tests locally using Xcode 12:
    1. Make sure command line is using Xcode 12: `sudo xcode-select --switch /Applications/Xcode_12_path.app`
    1. `npm run native test:e2e:ios:local`
